app/models/metric/statistic.rb

```diff
@@ -0,0 +1,13 @@
+module Metric::Statistic
+  def self.calculate_stat_columns(obj, attrs, ts = nil)
+    return {} unless obj.kind_of?(ExtManagementSystem)
```

Comment:
@zeari that would add the stats for other entities as well (services, projects, etc.) which could be interesting. Also it's nicer code-wise :+1: